### PR TITLE
Controller spec

### PR DESCRIPTION
**Pull request by alexander16108**
For this PR:

-   Add RSpec as a gem to your project.
-   Create a Request spec file for all your controllers.
-   Make sure that for each action you check:
    - If response status was correct.
    - If a correct template was rendered.
    - If the response body includes correct placeholder text.

### CODE REVIEW

File: spec/requests/post_spec.rb [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+    end
+
+    it 'should render correct template' do
+      expect(response).to render_template(:show)
+    end
+
+    it 'should have correct placeholder' do
+      expect(response.body).to include('This page displays the users details')
+    end
+  end
```
Comment (Lameck1) [inline spec/requests/post_spec.rb]: - [ ] All these tests are an exact copy of the Users Controller request tests and they do not test the Posts controller in any way. Please consider writing tests for the Posts controller.

File: spec/requests/user_spec.rb [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+      expect(response).to have_http_status(200)
+    end
+
+    it 'should render correct template' do
+      expect(response).to render_template(:show)
+    end
+
+    it 'should have correct placeholder' do
+      expect(response.body).to include('This page displays the users details')
+    end
```
Comment (Lameck1) [inline spec/requests/user_spec.rb:33]: - [ ] This test is failing. Please consider refactoring. The failure screenshot is shown below:

  
image

**Review by Lameck1 — CHANGES_REQUESTED** [review]
Hi, @alexander16108, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by DeliceLydia — APPROVED** [review]
# Congratulations🎉🎉 
 

## Hi @alexander16108 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



### Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard).. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**